pkg/cli/start.go

```diff
@@ -1042,6 +1042,9 @@ func doShutdown(ctx context.Context, c serverpb.AdminClient, onModes []int32) er
 	// out, or perhaps drops the connection while waiting). To that end, we first
 	// run a noop DrainRequest. If that fails, we give up.
 	if err := checkNodeRunning(ctx, c); err != nil {
+		if grpcutil.IsClosedConnection(err) {
+			return nil
```

Comment:
I think if you try to run `cockroach quit` against a node that's not running, we still want to fail instead of swallowing the error.

Reply:
The thing still fails. `IsClosedConnection` does not return true on ECONNREFUSED.